types/supercluster/index.d.ts

```diff
@@ -68,31 +70,29 @@ declare class Supercluster {
 /**
  * A very fast JavaScript library for geospatial point clustering for browsers and Node.
  */
-declare function supercluster(options: Options): Supercluster;
-declare namespace supercluster {
-    type Point = GeoJSON.Feature<GeoJSON.Point>;
-    type Points = Point[];
-    type Clusters = Cluster[];
-    type TileFeatures = TileFeature[];
-    type BBox = [number, number, number, number];
-    interface ClusterProperties {
-        cluster?: boolean;
-        cluster_id?: number;
-        point_count?: number;
-        point_count_abbreviated?: number;
-        sum?: number;
-        [key: string]: any;
-    }
-    interface Cluster extends Point {
-        properties: ClusterProperties;
-    }
-    interface TileFeature {
-        type: 1;
-        geometry: Array<[number, number]>;
-        tags: ClusterProperties;
-    }
-    interface Tile {
-        features: TileFeatures;
-    }
+export default function supercluster(options: Options): Supercluster;
```

Comment:
`supercluster` doesn't export to default... 🤔 Don't think this will work when you execute the compiled Typescript code.

https://github.com/mapbox/supercluster/blob/master/index.js#L5

Reply:
Ah yes you are right, I missed it, I actually need to import the interface `Supercluster` in my app and couldn't figure out how to do it with the current types, any suggestions?

Reply:
This should now work on the next patch/minor release.
https://github.com/mapbox/supercluster/pull/62

Reply:
This section is 👍